src/connection_manager.c

```diff
@@ -745,12 +757,39 @@ QD_EXPORT qd_connector_t *qd_dispatch_configure_connector(qd_dispatch_t *qd, qd_
 
     qd_error_clear();
 
-    if (ct && load_server_config(qd, &ct->config, entity, false) == QD_ERROR_NONE) {
+    if (ct && load_server_config(qd, &ct->config, entity, false, 0) == QD_ERROR_NONE) {
         ct->policy_vhost = qd_entity_opt_string(entity, "policyVhost", 0); CHECK();
         DEQ_ITEM_INIT(ct);
         DEQ_INSERT_TAIL(cm->connectors, ct);
         log_config(cm->log_source, &ct->config, "Connector");
 
+        //
+        // If this connection has a data-connection-group, set up the group members now
+        //
+        if (ct->config.has_data_connectors) {
+            qd_generate_discriminator(ct->group_correlator);
```

Comment:
How would you feel about adding the router node id as part of the group_correlator value?  For debug trace of Open frames and probably prevent very unlikely naming collisions?

Reply:
N.M.  Looks like a fixed size is necessary for allocation the correct sized tracking arrays, etc.